seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/utils/RowConverter.java

```diff
@@ -381,9 +386,13 @@ public static InternalRow convert(
                     break;
                 case DATE:
                     LocalDate date = (LocalDate) seaTunnelRow.getField(i);
-                    LocalTime time = LocalTime.of(0, 0, 0);
-                    binaryWriter.writeTimestamp(
-                            i, Timestamp.fromLocalDateTime(date.atTime(time)), 3);
+                    String dateStr = date.format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
```

Comment:
Wouldn't it be more convenient to directly obtain the number of days from LocalDate?